Sources/TuistGenerator/Mappers/AutogeneratedProjectSchemeWorkspaceMapper.swift

```diff
@@ -86,13 +85,28 @@ public final class AutogeneratedProjectSchemeWorkspaceMapper: WorkspaceMapping {
                 targets: testableTargets,
                 arguments: nil,
                 configurationName: project.defaultDebugBuildConfigurationName,
-                coverage: codeCoverage,
-                codeCoverageTargets: [],
+                coverage: coverageSettings.isEnabled,
+                codeCoverageTargets: coverageSettings.targets,
                 expandVariableFromTarget: nil,
                 preActions: [],
                 postActions: [],
                 diagnosticsOptions: [.mainThreadChecker]
             )
         )
     }
+
+    private func codeCoverageSettings(workspace: WorkspaceWithProjects) -> (isEnabled: Bool, targets: [TargetReference]) {
+        let codeCoverageTargets = workspace.workspace.codeCoverageTargets(
+            mode: codeCoverageMode,
+            projects: workspace.projects
+        )
+
+        switch codeCoverageMode {
+        case .all: return (true, codeCoverageTargets)
```

Comment:
codeCoverageTargets is empty here, is it expected?
if yes, I find it a bit confusing that we should enable it on all, `targets` is empty

Reply:
well it is the same as is for `TestAction` with enabled code coverage and empty code coverage targets...